CMakeLists.txt

```diff
@@ -20,6 +20,15 @@ set_property(TARGET boost PROPERTY
 set_property(TARGET boost PROPERTY
     INTERFACE_LINK_LIBRARIES ${Boost_LIBRARIES})
 
+# Include Folly as an imported target
+find_package(Folly 2017.07.31.00 REQUIRED)
+add_library(folly INTERFACE IMPORTED)
+set_property(TARGET folly PROPERTY
+    INTERFACE_INCLUDE_DIRECTORIES ${FOLLY_INCLUDE_DIR})
+set(FOLLY_LINK_FLAGS "-lglog")
```

Comment:
This flag specifies that [glog](https://github.com/google/glog) should also be linked when linking Folly. Without this flag, I ran into a build error on OSX indicating that glog wasn't found.